runtime/sema/checker.go

```diff
@@ -937,40 +937,45 @@ func CheckIntersectionType(
 
 		// The intersections may not have clashing members
 
-		// TODO: also include interface conformances' members
-		//   once interfaces can have conformances
+		checkClashingMember := func(interfaceType *InterfaceType) {
```

Comment:
How does this deal with the delta problem? e.g: having `{A, B}` when `interface A: C {}` and `interface B: C {}` and `C` has a member `foo`.

Reply:
Good point, it needs to handle that as well

Reply:
The function already handled it. It does not matter if the member is inherited or not, all that is checked is that the all members of all interfaces have the same type.

Added a test in cc6258f5267be7344341b791ca60d9bd6ee00554